native/cocos/renderer/gfx-agent/DeviceAgent.cpp

```diff
@@ -141,6 +141,9 @@ void DeviceAgent::acquire(Swapchain *const *swapchains, uint32_t count) {
 }
 
 void DeviceAgent::present() {
+    if (!cc::gfx::Device::getInstance()->isRendererAvailable()) {
```

Comment:
I think should disable rendering in render pipeline, may be in root.cpp?

Reply:
version of v3.5.0, root is not native